### DISCUSSION TITLE
Deployment to BSC and Klaytn

### DISCUSSION THREAD

**Discussion by pamungkaski** [ProjectOpenSea/seaport#585] [Q&A] [answered]
Hey guys, wanted to know when would be the deployment for Klaytn and BSC will be done? or can we (the people outside opensea) actually deploy it our selves using the guide here https://github.com/ProjectOpenSea/seaport/blob/main/docs/Deployment.md ?

Thanks!

**Comment by Slokh** [comment, marked as answer]
Anyone can deploy it to the same Seaport address using those steps! If you’re up for the challenge, go for it. Be aware that a lot of networks have various configuration issues that may block you though (i.e. Klaytn which we’re trying to work through)

**Reply by arr00** [reply]
Seaport has been deployed to BSC https://bscscan.com/address/0x00000000006c3852cbEf3e08E8dF289169EdE581#code

**Reply by pamungkaski** [reply]
oh niceee
